tekton/cd/chains/overlays/dogfooding/chains-config.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: chains-config
+  namespace: tekton-chains
+data:
+  artifacts.oci.signer: kms
+  artifacts.oci.storage: oci
+  artifacts.taskrun.format: in-toto
+  artifacts.taskrun.signer: kms
+  artifacts.taskrun.storage: oci
+  signers.kms.kmsref: gcpkms://projects/tekton-releases/locations/global/keyRings/chains/cryptoKeys/signing-key
```

Comment:
Though it seems to have worked earlier, I think this should be 
gcpkms://projects/tekton-releases/locations/global/keyRings/chains/cryptoKeys/signing-key/cryptoKeyVersions/1